packages/vx-legend/src/legends/Legend.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import LegendItem from './LegendItem';
+import LegendLabel from './LegendLabel';
+import LegendShape from './LegendShape';
+import labelQuantile from '../labels/quantile';
+
+Legend.propTypes = {
+  className: PropTypes.string,
+  style: PropTypes.object,
+  scale: PropTypes.func.isRequired,
+  labels: PropTypes.arrayOf(
+    PropTypes.shape({
+      extend: PropTypes.array,
+      text: PropTypes.string,
+      value: PropTypes.any,
+    })
+  ),
+  shapeWidth: PropTypes.oneOfType([
+    PropTypes.number,
+    PropTypes.string,
+  ]),
+  shapeHeight: PropTypes.oneOfType([
+    PropTypes.number,
+    PropTypes.string,
+  ]),
+  shapeMargin: PropTypes.string,
+  labelMargin: PropTypes.string,
+  itemMargin: PropTypes.string,
+  direction: PropTypes.string,
+  itemDirection: PropTypes.string,
+};
+
+export default function Legend({
+  className,
+  style,
+  scale,
+  shape,
+  labels,
+  shapeWidth = 15,
+  shapeHeight = 15,
+  shapeMargin = '2px 4px 2px 0',
+  labelAlign = 'left',
+  labelMargin = '0 4px',
+  itemMargin = '0',
+  direction = 'column',
+  itemDirection = 'row',
+  ...restProps,
+}) {
+  return (
+    <div
+      className={cx('vx-legend', className)}
+      style={style}
+      style={{
```

Comment:
not intentional, i've got this fixed locally, just haven't pushed it yet. Good catch! 👍